scripts/generateVersion.sh

```diff
@@ -1,13 +1,13 @@
 #!/bin/sh
 cp -fr ./src/version.h.in ./src/version.h
 
-HASH=$(git rev-parse HEAD)
-BRANCH=$(git branch --show-current)
-MESSAGE=$(git show ${GIT_COMMIT_HASH} | head -n 5 | tail -n 1 | sed -e 's/#//g' -e 's/\"//g')
```

Comment:
GIT_COMMIT_HASH was always unpopulated, it worked because it assumes HEAD when given no arguments.

Reply:
I'd **highly** recommend to get nix to set  _at least_ GIT_COMMIT_HASH, otherwise hyprpm will not work.

Reply:
the nix build is setting the git hash, but the variable name is actually HASH whereas the script was using GIT_COMMIT_HASH here (which is undefined in the nix or regular build). I could have fixed this by replacing `${GIT_COMMIT_HASH}` with `$HASH`, but there's no point in doing that since we just want to use HEAD anyway (which is the default for git show if no argument is passed).